packages/create-sitecore-jss/src/templates/nextjs/src/lib/page-props-factory/plugins/normalMode.ts

```diff
@@ -22,6 +22,12 @@ function extractPath(params: ParsedUrlQuery | undefined): string {
     path = '/' + path;
   }
 
+  // Remove SegmentId part from path, otherwise layout service will not find layout data
```

Comment:
Another one out of scope for this task, but we'll want to avoid duplicating all of the `normalMode.ts` just for this bit. I'll add something to the backlog.